### PR TITLE
week16 assignment

### PR DESCRIPTION
**Pull request by up2code3**
test directory = test in use
test store directory = test not in use

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ require("dotenv").config();
 
 const app = express();
 
+//old way of parsing
+//app.use(require("body-parser").urlencoded({ extended: true}));
+
+//new way of parsing
+app.use(express.urlencoded({ extended: true }));
```
Comment (jsuoz121) [inline app.js:25]: Nice!

File: utils/seed_db.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    let testUser = null;
+    try {
+        const mongoURL = process.env.MONGO_URI_TEST;
+        await Job.deleteMany({}); //deletes all job records
+        await User.deleteMany({}); // deletes all user records   
+//create a bew user
+        testUser = await factory.create("user", {password: testUserPassword });
+        console.log("seed_db: the new user" , testUser)
+//cerate 20 job entries linked to user above
+        const jobs = await factory.createMany("job", 3, { createdBy: testUser._id });
```
Comment (jsuoz121) [inline utils/seed_db.js:44]: Guessing changes to 3 for testing purposes but all good!

**Review by jsuoz121 — APPROVED** [review]
Great job

File: utils/seed_db.js (hunk 2 above)
Comment (up2code3) [reply utils/seed_db.js:44]: yes , But forgot to change the comment.

**Pull request merged**